.github/workflows/generate-swagger.yml

```
@@ -0,0 +1,233 @@
+  push:
+    branches:
+      - master
+      - np-integrate-open-api-spex
+    paths-ignore:
+      - "CHANGELOG.md"
+      - "**/README.md"
+      - "docker/*"
+      - "docker-compose/*"
+  workflow_dispatch:
+  release:
+      types: [published]
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 4 but found 6

(indentation)